.github/workflows/build-images.yml

```
@@ -29,8 +29,18 @@ jobs:
         uses: docker/setup-buildx-action@v2
         with:
           install: true
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        if: (github.ref != 'refs/heads/master' && github.event_name != 'pull_request') || github.event_name == 'schedule'
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        if: (github.ref != 'refs/heads/master' && github.event_name != 'pull_request') || github.event_name == 'schedule'
+        with:
+          registry: ghcr.io
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Hmm you hope the username for GHCR is the same as GitHub's? But yeah may actually be the case, just wonder if there is a `github.username`?

---

it needs to be lowercase and as you see the organization name is not in lowercase

---

We had to recently change the docker hub account, because they converted our former account into a (sponsored, open source) organization account. The new account is a [bot ID](https://hub.docker.com/u/privatebinbot) that is not otherwise used on github.

Reading up on the [actions doc](https://github.com/docker/login-action#github-container-registry), do we maybe need to use this?

```suggestion
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```

---

```suggestion
          username: privatebin
```